### PR TITLE
UI of ChatActivity

### LINKED ISSUE #90: UI IMPROVEMENT : Chat Activity
Make the UI a little more pleasant looking and add some changes like repositioning the send image button and add new default ic_male and ic_female icons and redesign the chat text view

### PR DESCRIPTION
**Pull request by riturajjain2000**
Fixes #90

### CODE REVIEW

File: app/src/main/res/layout/activity_chat.xml [hunk 1]
```diff
@@ -110,7 +110,10 @@
                         android:layout_height="wrap_content"
                         android:layout_centerVertical="true"
                         android:layout_marginStart="45dp"
-                        android:layout_marginEnd="40dp"
+
```
Comment (plazzy99) [inline app/src/main/res/layout/activity_chat.xml:113]: remove extra line spacing

File: app/src/main/res/layout/message_list_other.xml [hunk 2]
```diff
@@ -5,19 +5,17 @@
...
-        android:layout_marginEnd="80dp"
-        app:cardBackgroundColor="#129A7F"
-        android:layout_marginStart="4dp"
-        app:cardCornerRadius="5dp">
 
-        <androidx.constraintlayout.widget.ConstraintLayout
-            android:layout_width="match_parent"
-            android:layout_height="match_parent">
+
+        <androidx.constraintlayout.widget.ConstraintLayout            android:background="@drawable/bg_receiver"
```
Comment (plazzy99) [inline app/src/main/res/layout/message_list_other.xml:10]: refractor code

**Review by plazzy99 — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/message_list_other.xml (hunk 2 above)
Comment (riturajjain2000) [reply app/src/main/res/layout/message_list_other.xml:10]: > please share screen shots whenever making pr on UI

OK

**Comment by riturajjain2000** [comment]
Screenshot_1609861918

**Comment by riturajjain2000** [comment]
redesigned the chat list and shifted the send image icon

**Review by plazzy99 — APPROVED** [review]
👍🏻

**Pull request merged**